### DISCUSSION TITLE
Google AMP

### DISCUSSION THREAD

**Discussion by darrenpaul** [nuxt/nuxt#16104] [Questions]
Does anyone have a guide for sorting out Google AMP and Nuxt 3 for a static generated site.
I am struggling to sort out these issues.

I have tried using '@nuxtjs/amp' but its causing build issues.

tried following this tutorial and others but with no success
https://hashinteractive.com/blog/how-to-use-nuxt-amp-module-to-create-accelerated-mobile-pages/

<img width="949" alt="Screenshot 2022-11-29 at 15 27 08" src="https://user-images.githubusercontent.com/27204436/204651411-f4c7a8f6-6486-4c58-854d-138a12ce0ee4.png">

**Comment by darrenpaul** [comment]
For anyone having this issue, making my site SSR fixed this issue. It was previously SSG

**Reply by anjali89r** [reply]
did you use the nuxt-amp module for nuxt 3. I'm also facing same issue and suggestions should be helpful.

**Reply by darrenpaul** [reply]
nuxt-amp for nuxt 3 wasn't around when I tried. I'll give it a try and see if it works now 🙂

**Reply by Tyraxaes** [reply]
Any news on this? I tried to install @nuxtjs/amp but is not working on installing the dependency.
Also the hooks provided by nuxt3 doesnt seem to accept something like after one page is rendered to edit the html to custom the html after is rendered and before is served to the client.

**Reply by gtomanini** [reply]
hi @darrenpaul 
you just set SSR? No issues with nuxt/entry.js on amp test?
